lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		reset!
+	end
+
+	def reset!
+		@cells = Array.new(9, " ")
+	end
+
+	def display
+		puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+		puts "-----------"
+		puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+		puts "-----------"
+		puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+	end
+	
+	def position(input)
+		self.cells[input.to_i-1]
+	end
+
+	def full?
+		if self.cells.all?{|cell| cell == "X" || cell == "O"}
+			true
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression.

```
def full?
  self.cells.all?{|cell| cell == "X" || cell == "O"}
end
```